share/pot/export.sh

```diff
@@ -26,21 +27,63 @@ export-help() {
 _export_pot()
 {
 	# shellcheck disable=SC3043
-	local _pname _dset _snap _tag _dir _file _clvl
+	local _pname _dset _snap _tag _check_tag _prev_tag _prev_snap
+	# shellcheck disable=SC3043
+	local  _dir _file _clvl _meta _origin _origin_pname_snapshot
+	# shellcheck disable=SC3043
+	local _origin_pname _origin_snapshot _origin_tag _highest_version
 	_pname="$1"
 	_snap="$2"
 	_tag="$3"
-	_dir="$4"
-	_clvl="$5"
+	_check_tag="$4"
+	_dir="$5"
+	_clvl="$6"
 	_file="${_dir}/${_pname}_${_tag}.xz"
 	_dset="${POT_ZFS_ROOT}/jails/$_pname"
+        _meta="-"
+
+	_prev_tag=$(zfs get -H :pot.tag "${_dset}" | awk '{ print $3 }')
```

Comment:
a more robust way to deal with the `zfs` cli is to use the `-o field` option. Sometimes the add/reorder/remove columns from the output, removing the need to use `awk`.
This line can be rewritten as:
`_prev_tag=$(zfs get -H -o value :pot.tag "${_dset}")`

Reply:
I tried "-o :pot.tag" and "-o pot.tag", wasn't aware of the "-o value" syntax, thanks!

=> will change